### PR TITLE
Switch to a blueprint/create_app pattern

### PR DESCRIPTION
**Pull request by ryan-lane**
This change updates how the app is created for initialization, for easier testing. A change needed to switch to blueprints is to no longer use `app.config['<var>']`, but instead to use `settings.<var>`. This change also updates the encrypted settings and allows access directly to the encrypted settings from the settings variables.

### CODE REVIEW

File: confidant/app.py [hunk 1]
```diff
@@ -1,33 +1,62 @@
 import logging
 
 import boto3
+import guard
```
Comment (skiptomyliu) [inline confidant/app.py]: I think we need to update pip requirements for this

Comment (skiptomyliu) [reply confidant/app.py]: mmh nevermind

File: confidant/app.py [hunk 2]
```diff
@@ -1,33 +1,62 @@
...
+def create_app():
+    static_folder = settings.STATIC_FOLDER
+
+    app = Flask(__name__, static_folder=static_folder)
+    app.config.from_object(settings)
+    app.config.update(settings.encrypted_settings.get_all_secrets())
+    app.debug = settings.DEBUG
+
+    if settings.SSLIFY:
+        sslify = SSLify(app, skips=['healthcheck'])  # noqa
```
Comment (skiptomyliu) [inline confidant/app.py]: does the var `sslify` need to be used elsewhere?  Looks like it's unreferenced at moment

Comment (ryan-lane) [reply confidant/app.py]: Ah, yeah, I could probably change this to just `SSLify(app, skips=['healthcheck'])` Let me update that.

**Pull request merged**